pkgs/development/tools/database/liquibase/default.nix

```diff
@@ -48,7 +48,10 @@ stdenv.mkDerivation rec {
     in
     ''
       mkdir -p $out
-      mv ./{lib,licenses,internal/lib/liquibase-core.jar,internal/lib/postgresql.jar,internal/lib/picocli.jar} $out/
+      mv ./{lib,licenses} $out/
+
+      mkdir -p $out/internal/lib
+      mv ./internal/lib/*.jar $out/internal/lib/
```

Comment:
Only jars?

Reply:
I tried to keep it as close to the original code as possible. There are only jars in that directory anyways. I could change it though if you think it's better. I don't mind.